docs/synthetics/http-monitor.md

```diff
@@ -27,35 +27,57 @@ Max length of the header, query param, and cookie name and value are 512 and 102
 By default, the HTTP monitor adds the headers below for all requests sent from the agent:
 
 | Name | Value | Description |
-|---|---|---|
-| `x-sematext-origin`  | `synthetics` | Origin of the request. Can be used to identify Synthetics requests in the back end or filter the requests in a firewall.  |
-| `x-sematext-synthetics-id`  | `<run-id>` | Uniquely identifies this request. Can be used for tracing and correlation in the back end. |
+| --- | --- | --- |
+| `x-sematext-origin` | `synthetics` | Origin of the request. Can be used to identify Synthetics requests in the back end or filter the requests in a firewall. |
+| `x-sematext-synthetics-id` | `<run-id>` | Uniquely identifies this request. Can be used for tracing and correlation in the back end. |
+
+## Authentication
+HTTP monitors support header-based authentication. Custom HTTP request headers can be specified when creating a monitor. To use Basic authentication, add a custom request header according to the following specifications:
+
+| Name | Value | Description |
+| --- | --- | --- |
+| `Authorization` | `Basic XYZ` | **XYZ** is the **Base64** encoding of the **username:password** string. |
+
+
+After **Base64** encoding the string `username:password` above, would become `dXNlcm5hbWU6cGFzc3dvcmQ=`. This is then prepended with `Basic` to get the final `Authorization` header value of `Basic dXNlcm5hbWU6cGFzc3dvcmQ=`.
+
+So if your username was `username` and you password was `password`, then your `Authorization` header would look as follows:
+
+| Name | Value |
+| --- | --- |
+| `Authorization` | `Basic dXNlcm5hbWU6cGFzc3dvcmQ=` |
+
+
+It is possible to get the **Base64** encoded value of your `username` and `password` by using the following command in a terminal, you should replace `username` with your username and `password` with your password respectively:
+```
+echo -n "username:password" | base64
+```
```

Comment:
In case somebody is reading but doesn't have terminal handy for some reason (mobile device?) maybe we could also point to some online Base64 encoders like https://www.base64encode.org/

Reply:
I was in two minds about this, as linking to external websites can sometimes be a bad idea in relation to SEO. I will add in the link.